external-builds/pytorch/run_linux_pytorch_tests.py

```diff
@@ -454,7 +463,12 @@ def main() -> int:
     print(f"Using PyTorch version: {pytorch_version}")
 
     # Get tests to skip
-    tests_to_skip = get_tests(amdgpu_family, pytorch_version, not args.debug)
+    tests_to_skip = get_tests(
+        amdgpu_family=amdgpu_family,
+        pytorch_version=pytorch_version,
+        platform=platform.system(),
+        create_skip_list=not args.debug,
+    )
 
     # Allow manual override of test selection
     if args.k:
```

Comment:
i never fixed that one. so if you feel motivated it should be 

```
if args.k:
   tests_to_skip = args.k
else:
    tests_to_skip = get_tests(...
```

Reply:
Ack. That can be a separate PR.